packages/augur-artifacts/src/index.ts

```diff
@@ -55,7 +56,7 @@ export interface NetworkContractAddresses {
 }
 
 export async function setAddresses(networkId: NetworkId, addresses: ContractAddresses): Promise<void> {
-  const filepath = `${__dirname}/addresses.json`;
+  const filepath = path.join(__dirname, "../src/addresses.json"); // be sure to be in src dir, not build
```

Comment:
should be overwriting source file addresses.json, not the one in the build dir... this was the bug